src/LayerPlanBuffer.cpp

```diff
@@ -203,8 +351,18 @@ void LayerPlanBuffer::insertPreheatCommands()
             continue;
         }
         
-        double avg_flow = extruder_plan.estimates.getMaterial() / time; // TODO: subtract retracted travel time
-        extruder_plan.required_temp = preheat_config.getTemp(extruder_plan.extruder, avg_flow, extruder_plan.is_initial_layer);
+        double avg_flow = extruder_plan.estimates.getMaterial() / time;
+        extruder_plan.printing_temperature = preheat_config.getTemp(extruder, avg_flow, extruder_plan.is_initial_layer);
```

Comment:
The flow-temp graph returns the standard printing_temperature if flow based temp is disabled....
So the preheat_config already takes care of this.